src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -700,14 +713,27 @@ private static boolean segmentIsWithinRange(RepairSegment segment, RingRange ran
 
   private static RepairSegment createRepairSegmentFromRow(Row segmentRow) {
 
-    RepairSegment.Builder builder = RepairSegment.builder(
+    List<RingRange> tokenRanges =
+        JsonParseUtils.parseRingRangeList(
+            Optional.fromNullable(segmentRow.getString("token_ranges")));
+
+    io.cassandrareaper.core.Segment.Builder segmentBuilder = Segment.builder();
```

Comment:
does not need to be fully qualified?

Reply:
It does, otherwise it conflicts with another Builder class.

Reply:
can you change the line to 
```
Segment.Builder segmentBuilder = Segment.builder();
```